common/kokkos-sampler/Makefile

```diff
@@ -0,0 +1,19 @@
+CXX=g++
+CXXFLAGS=-O3 -std=c++11 -g \
+	-I$(OVIS_DIR)/include/ -I./include
```

Comment:
```suggestion
CXXFLAGS=-O3 -std=c++11 -g
```

Reply:
Thank you for pointing that out. They've been deleted now.